docs/source/collect/run.md

```diff
@@ -60,58 +60,41 @@ spec:
         imagePullPolicy: IfNotPresent
 
 ```
-## Examples using `imagePullSecret`
+## Examples using private images with `imagePullSecret`
 
-### Using dockerconfigjson secrets
-
-In order to use dockerconfigjson secrets, `type: kubernetes.io/dockerconfigjson` must be added to the specs. It only accepts one argument, `.dockerconfigjson`, in data field. 
-Either a valid path to a docker config.json file, or a base64 encoded config.json file must be provided. An option to encode the file in macOS or Linux based systems is running the following command in the console (if base64 is installed): 
+### Using a preexisting secret
 
-```shell
-   cat path_to_file/config.json | base64
-```
-Further info about config.json file and .dockerconfigjson secrets may be found here: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+If the secret used to pull the image exists already in the cluster, you can use it by providing the run colector with the name of such secret.
 
 ```yaml
 spec:
   collectors:
-     - run
-         collectorName: "run-ping"
-         image: busybox:1
-         namespace: default
-         imagePullSecret:
-            name: mysecret
-            data: 
-              .dockerconfigjson: path/to/file/config.json
-            type: kubernetes.io/dockerconfigjson
-```
-```yaml
-spec:
-  collectors:
-     - run
+     - run:
          collectorName: "run-ping"
          image: busybox:1
          namespace: default
```

Comment:
Let's make it a complete, working example.
```suggestion
         namespace: default
         command: ["ping"]
         args: ["-w", "5", "www.google.com"]
         imagePullPolicy: IfNotPresent
```

Reply:
Or use a different image, like `image: privateimage`, but you should still provide an example `command`, even if it doesn't really work.

Reply:
Same for the example below.